presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -302,147 +306,6 @@ public CallExpression comparison(OperatorType operatorType, RowExpression left,
         return call(operatorType.getOperator(), functionHandle, left.getType(), left, right);
     }
 
-    public class AggregationBuilder
-    {
-        private final TypeProvider types;
-        private PlanNode source;
-        private Map<VariableReferenceExpression, Aggregation> assignments = new HashMap<>();
-        private AggregationNode.GroupingSetDescriptor groupingSets;
-        private List<VariableReferenceExpression> preGroupedVariables = new ArrayList<>();
-        private Step step = Step.SINGLE;
-        private Optional<VariableReferenceExpression> hashVariable = Optional.empty();
-        private Optional<VariableReferenceExpression> groupIdVariable = Optional.empty();
-        private Session session = testSessionBuilder().build();
```

Comment:
I see a lot of unrelated changes in this PR. We need to leave them untouched. Given there are too many of them, we could either create a new PR or clean up this one. Feel free to pick whichever one is easier. Given the `WarningCollector` move will cause a lot of file changes, also feel free to do it first (either a new PR or a new commit in a PR). Then, we can follow with the actual change.